.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
Please add all the files for linter from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) config